_tools/cite-guide.md

```diff
@@ -0,0 +1,459 @@
+
+{% include alert-info.html heading="Testing requests can be submitted to the FPKI Technical Working Group at fpki-ttips@listserv.gsa.gov. All testing requests should be submitted ten business days in advance." %} <br>
+
+For unscheduled testing, the CITE and FPKI Partner repositories are internet accessible and available for testing (including vendors and other Relying Parties). Unscheduled testing may be conducted at any time if the below is true.
+1. All parties involved agree to provide the necessary support; or
+2. The testing party does not need support from any other CITE Participant (in which case, the testing party is willing to accept that services may or may not be available). <br>
+
+## Repository Availability
+
+CITE Participant repositories should be available during regular business hours for scheduled and unscheduled testing. Each CITE Participant should leave its repository services operational and available 24 hours a day, 7 days a week. CITE Participants should follow the below table on repository availability requirements.
+
+| Days | Time | Description | Repository Availability Requirement |
+| --------- | --------------- | ------------------ | ----------------------------------- |
+| Mon - Fri | 0900 - 1700 EST | Business Hours     | CITE Participant repository services should be operational and available, except for scheduled downtime and federal holidays. |
+| Mon - Fri | 1700 - 0900 EST | Non-Business hours | No requirement |
+| Sat - Sun | 0001 - 0000 EST | Non-Business hours | No requirement |
+
+{% include alert-warning.html content="CITE Test Common Policy and Test Federal Bridge, at a minimum, will provide 20% availability per month. CITE should not be used or relied upon for near-production availability." %} <br>
+
+## Technical Support Availability
+
+CITE Participants shall provide the FPKI Technical Working Group with email and phone information for at least two technical contactsto help coordinate any technical service issues. In lieu of providing individual names for technical POCs, CITE Participants may establish a group or other organizational-based email addresses for communications with the appropriate technical contacts. This information will only be made available (in a controlled manner) to CITE Participants, FPKI Applicants (if applicable), and vendors supporting the FPKI as needed during testing or troubleshooting. CITE Participants involved in scheduled testing shall provide the issuance, management, and troubleshooting necessary to help resolve any issues.
+
+{% include alert-warning.html content="CITE Participant technical support is only available for scheduled testing with any outage resolved on a best effort basis. " %} <br>
+
+## Test Websites

```

Comment:
If there is none - don't list at all.

Reply:
I like to keep it as a placeholder if an affiliate wants to share one, we know where to put it.